src/app/main/component/events/components/create-edit-events/create-edit-events.component.ts

```diff
@@ -192,29 +198,36 @@ export class CreateEditEventsComponent implements OnInit, OnDestroy {
     if (this.editMode) {
       sendEventDto = {
         ...sendEventDto,
-        imagesTodelete: this.imagesToDelete
+        imagesToDelete: this.imagesToDelete,
+        additionalImages: this.oldImages.slice(1),
+        id: this.editEvent.id,
+        titleImage: this.oldImages[0]
       };
     }
-    const test = true;
-    if (this.checkdates && this.eventFormGroup.valid && tagsArr.length && test) {
+
+    if (this.checkdates && this.eventFormGroup.valid && tagsArr.length) {
       this.checkAfterSend = true;
       const formData: FormData = new FormData();
       const stringifiedDataToSend = JSON.stringify(sendEventDto);
-      formData.append('addEventDtoRequest', stringifiedDataToSend);
+      let dtoName = 'addEventDtoRequest';
+      if (this.editMode) {
```

Comment:
don't use if here
let dtoName =this.editMode ?  'eventDto' : 'addEventDtoRequest';